src/js/components/Video/Video.js

```diff
@@ -80,6 +80,26 @@ const Video = forwardRef(
     const containerRef = useRef();
     const scrubberRef = useRef();
     const videoRef = useForwardedRef(ref);
+    const [newControls, setNewControls] = useState({
+      position: 'over',
+      items: ['volume', 'reduceVolume', 'fullScreen'],
+    });
+
+    useLayoutEffect(() => {
```

Comment:
I don't think this needs to use `useLayoutEffect()`. There's nothing about the contents that depends on the DOM render state.
```suggestion
    useEffect(() => {
```